File: 1672-richest-customer-wealth/1672-richest-customer-wealth.java
class Solution {
    public int maximumWealth(int[][] accounts) {
        int answer=Integer.MIN_VALUE;
        for(int[] account : accounts){
            int tempSum=0;
            for(int money : account){
                tempSum+=money;
            }
            answer=Math.max(answer,tempSum);
        }
        
        return answer;
    }
}

File: 1672-richest-customer-wealth/1672-richest-customer-wealth.go
func maximumWealth(accounts [][]int) int {
	ret := math.MinInt32

	for _, account := range accounts {
		tempSum := sumOfArray(account)
		if tempSum > ret {
			ret = tempSum
		}
	}

	return ret
}

func sumOfArray(array []int) int {
	sum := 0
	for _, val := range array {
		sum += val
	}
	return sum
}